src/Editor/Client/Components/EditorPage.razor

```diff
@@ -464,11 +474,11 @@
                 },
             }
         },
-    
-    
-    
-    
-    
+
+
+
+
+
```

Comment:
So these were white spaces, lets just clean this up

Reply:
Sounds good